openreview/venue_request/venue_request.py

```diff
@@ -715,18 +715,24 @@ def setup_decision_stage(self):
                 'required': True
             },
             'decision_options': {
-                'description': 'What are the decision options (provide comma separated values, e.g. Accept (Best Paper), Accept, Reject)? Leave empty for default options - "Accept (Oral)", "Accept (Poster)", "Reject"',
+                'description': 'List all decision options. By default, decisions containing "Accept" signify acceptance to the venue. Otherwise, please specify the accept options in "Accept Decision Options". All other decisions will be treated as a rejection. Provide comma separated values, e.g. "Accept (Best Paper), Invite to Archive, Reject". Leave empty for default options - "Accept (Oral)", "Accept (Poster)", "Reject"',
                 'value-regex': '.*',
                 'order': 30
             },
```

Comment:
What do you think about changing the description to just:

> List all decision options. Provide comma separated values, e.g. "Accept (Best Paper), Invite to Archive, Reject". Leave empty for default options - "Accept (Oral)", "Accept (Poster)", "Reject"

I'm worried it's too confusing and PCs would fill out the fields incorrectly. For example, if "Accept" and "Invite to Archive" are considered accept options, they may just put "Invite to Archive" in `accept_decision_options`. I think this would cause even "Accept" papers to be rejected since it's not listed as an accept option. We could also add to the `accept_decision_options` description something like:

> Leaving this empty will treat only decisions with "Accept" as accepted.

Reply:
can we make the field mandatory and have a default value?

Reply:
We do use ` ['Accept (Oral)', 'Accept (Poster)', 'Reject']` as the default options in the request form if they leave the field empty. Do you want this value to be added to the request form?

Reply:
yes, I think it is more clear for the users to know which are the default options how they can edit them.

Reply:
Yes I think with the simplified description we don't need to make the fields required. I fixed it and added the default value. I think it's much better now.